src/main/java/Asuka.java

```diff
@@ -0,0 +1,256 @@
+    }
+
+    public void printTask() {
+        if (this.isDone) {
+            System.out.println("[X] " + this.description);
+        } else {
+            System.out.println("[ ] " + this.description);
+        }
+    }
+}
+
+class Todo extends Task {
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    public void printTask() {
+        if (this.isDone) {
+            System.out.println("[T][X] " + this.description);
+        } else {
+            System.out.println("[T][ ] " + this.description);
+        }
+    }
+}
+
+class Deadline extends Task {
+
+    public Deadline(String description) {
+        super(description);
+    }
+
+    public void printTask() {
+        if (this.isDone) {
+            System.out.println("[D][X] " + this.description);
+        } else {
+            System.out.println("[D][ ] " + this.description);
+        }
+    }
+}
+
+class Event extends Task {
+
+    public Event(String description) {
+        super(description);
+    }
+
+    public void printTask() {
+        if (this.isDone) {
+            System.out.println("[E][X] " + this.description);
+        } else {
+            System.out.println("[E][ ] " + this.description);
+        }
+    }
+}
+
+enum Commands {
+    list, mark, unmark, add, bye
+}
+
+public class Asuka {
+
+    public static final int MAX_TASKS = 100;
+
+    public static void main(String[] args) {
+        // Welcome message
+        printWelcomeMessage();
+
+        //Initialise
+        Commands command = null;
+        int taskCount = 0;
+        Task[] tasks = new Task[MAX_TASKS];
+        Scanner myObj = new Scanner(System.in);
+
+        //Loop through different commands, until "bye" command
+        while (command != Commands.bye) {
+
+            // Read input
+            String input = myObj.nextLine();
+            String[] inputs = input.split(" ");
+
+            // Determine command
+            command = getCommands(input, inputs);
+            if (command == null) {
+                printInvalidCommand();
+                continue;
+            }
+
+            // Execute command
+            switch (command) {
+            case mark:
+                mark(inputs, taskCount, tasks);
+                break;
+            case unmark:
+                unmark(inputs, taskCount, tasks);
+                break;
+            case list:
+                list(taskCount, tasks);
+                break;
+            case add:
+                taskCount = updateTaskCount(input, tasks, taskCount);
+                break;
+            case bye:
+                bye();
+                myObj.close();
+                myObj = null;
+                break;
+            default:
+                break;
+            }
+        }
+    }
+
+    private static int updateTaskCount(String input, Task[] tasks, int taskCount) {
+        String[] inputs;
+        System.out.println("____________________________________________________________");
```

Comment:
I suggest declaring a constant instead of a long string of underscores.
private static String LINE = "___________________________________________";
and just print LINE where you need it.